test/node/multipart.js

```diff
@@ -177,6 +177,20 @@ describe('Request', function(){
         done();
       })
     })
+    it('filesystem errors should be caught', function(done){
+      var timestamp = Date.now()
+      request
+          .post(base + '/echo')
+          .attach('filedata', 'test/node/fixtures/lorem-' + timestamp + '.ext')
```

Comment:
If this is supposed to be a non-existent file, let's make it simple and obvious and just use `'test/node/fixtures/non-existent-file.ext'`. The timestamp implies something about uniqueness that I don't think is really at play here.